frontend/app/index.tsx

```diff
@@ -2,9 +2,8 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { ThemeProvider } from '@mui/material';
 
-import theme from '../theme';
-
-import Home from './Home';
+import Home from './components/Home';
+import theme from './theme';
 
 ReactDOM.render(
   (
```

Comment:
Note also that this file was also moved out of `components` into `app`, since it's not exporting any components.